packages/backend/src/graphql/schema.graphql

```diff
@@ -405,6 +405,7 @@ input RegisterConnectionInput {
 input CreateFlowInput {
   triggerAppKey: String
   connectionId: String
+  flowName: String
```

Comment:
can make this `String!` when you enable name your pipe modal outside of empty state

Reply:
did this in the next PR! thanks